### ISSUE TITLE
Feature Request: Meal Plan entries with Recipe/Product names

### ISSUE THREAD

**Issue by zsarnett** [custom-components/grocy#45]
Ability to view the Meal Plan entries in a Calendar like way would be a great addition to this sensor. 

https://github.com/grocy/grocy/issues/777

**Comment by pironic** [comment]
doesn't even have to be a calendar tbh... 
![image](https://user-images.githubusercontent.com/541154/85082598-a6c9c400-b18c-11ea-8282-921228d1ebf6.png)
This is from a google keep card that my wife and I use. making a card like this in HA is the only thing holding us back from moving to grocy for meal planning.

**Comment by SebRut** [comment]
_Hey! Just wanted to let you know that I finally accepted, that I'll not be able to maintain this project for the foreseeable future. I'd be happy to hand most of this over to someone willing to work on this integration and will try to support this by working on SebRut/pygrocy._

**Comment by nervetattoo** [comment]
I just decided to try out Grocy and found the same lack in capabilities a surprise. This is the number one thing I want; Create a dashboard with the meal plan. So I decided to see if I can add this. PR for pygrocy is up at https://github.com/SebRut/pygrocy/pull/109

As soon as that is ironed out, merged and released, I'll look into adding it to the custom component. Years pass between each time I touch python so at this point I'm not interesting in taking over the project @SebRut , but I hope the PR is up to your standards though and that you have time to review and merge/release it.
